### PR TITLE
doc: fixed instruction in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Nereuxofficial**
clippy.sh is not in the root, so the command was not quite working.

Should there also be a note about the Powershell scripts for potential windows developers?

### CODE REVIEW

**Comment by tokatoka** [comment]
yes

**Comment by Nereuxofficial** [comment]
Done :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ For bugs, feel free to open issues or contact us directly. Thank you for your su
 
 ## Pull Request guideline
 
-Even though we will gladly assist you in finishing up your PR, try to
+Even though we will gladly assist you in finishing up your PR, try to:
+
 - keep all the crates compiling with *stable* rust (hide the eventual non-stable code under [`cfg`s](https://github.com/AFLplusplus/LibAFL/blob/main/libafl/build.rs#L26))
 - run `cargo +nightly fmt` on your code before pushing
-- check the output of `cargo clippy --all` or `./clippy.sh`
+- check the output of `cargo clippy --all` or `./scripts/clippy.sh` (On windows use `./scripts/clippy.ps1`)
```
Comment (tokatoka) [inline CONTRIBUTING.md:11]: .\scripts\clippy.ps1 ?

Comment (Nereuxofficial) [reply CONTRIBUTING.md:11]: I was under the impression that both work on Windows at least in Powershell but \ is probably better, yes

**Comment by tokatoka** [comment]
thanks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,15 +4,16 @@ For bugs, feel free to open issues or contact us directly. Thank you for your su
 
 ## Pull Request guideline
 
-Even though we will gladly assist you in finishing up your PR, try to
+Even though we will gladly assist you in finishing up your PR, try to:
+
 - keep all the crates compiling with *stable* rust (hide the eventual non-stable code under [`cfg`s](https://github.com/AFLplusplus/LibAFL/blob/main/libafl/build.rs#L26))
 - run `cargo +nightly fmt` on your code before pushing
-- check the output of `cargo clippy --all` or `./clippy.sh`
+- check the output of `cargo clippy --all` or `.\scripts\clippy.sh` (On windows use `./scripts/clippy.ps1`)
```
Comment (tokatoka) [inline CONTRIBUTING.md:11]: no wait it's the opposite

Comment (tokatoka) [reply CONTRIBUTING.md:11]: @Nereuxofficial 
can you fix it again?

Comment (Nereuxofficial) [reply CONTRIBUTING.md:11]: Oh my god. My apologies. Yes absolutely

**Pull request merged**